docs/src/pages/overlayEntry.mdx

```diff
@@ -0,0 +1,50 @@
+import PropertyTable from '../components/propertyTable';
+import overlayEntrySchema from '../../../api/components/overlayEntry.schema.json';
+
+# OverlayEntry
+
+This component is used to create an **overlay** that shows itself over the current application's UI. For example, it might be interesting to use it when displaying errors in a snackbar.
+
+This component uses the `showOverlay` boolean property to know whether it should be shown to the screen or not. This property **should be used** to open/close the overlayEntry. The overlayEntry **should not be** directly removed from the JSON tree as it will not follow the right disposing process, the overlayEntry will end up stuck on the screen with no way to remove it.
+
+It is recommended to put overlays inside the root `flex` component of the application. An example is shown below.
+
+## Examples
+
+### Simple overlayEntry
+
```

Comment:
```suggestion
## Examples

### Simple overlayEntry

This component uses the `showOverlay` boolean property to know whether it should be shown to the screen or not. This property **should be used** to open/close the overlayEntry. The overlayEntry **should not be** directly removed from the JSON tree as it will not follow the right disposing process, the overlayEntry will end up stuck on the screen with no way to remove it.

```